graphql/image_test.go

```diff
@@ -38,3 +38,25 @@ func TestPackages(t *testing.T) {
 	require.NotNil(t, res[0])
 	assert.Equal(t, testPackage, utility.FromStringPtr(res[0].Name))
 }
+
+func TestToolchains(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestToolchains")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+	testToolchain := "golang"
+	ami := "ami-0f6b89500372d4a06"
+	image := model.APIImage{
+		AMI: &ami,
+	}
+	opts := thirdparty.ToolchainFilterOptions{
+		AMI:   ami,
+		Limit: 5,
+	}
+	res, err := config.Resolvers.Image().Toolchains(ctx, &image, opts)
+	require.NoError(t, err)
+	assert.Len(t, res, 5)
```

Comment:
I think we should `require` because `res[0]` will probably panic if the length is 0. Could we fix this for the `TestPackages` test too?